tools/alticonstool/BUILD

```diff
@@ -0,0 +1,21 @@
+licenses(["notice"])
+
+py_binary(
+    name = "alticonstool",
+    srcs = ["alticonstool.py"],
+    srcs_version = "PY2AND3",
```

Comment:
```suggestion
    srcs_version = "PY3",
```

I've just tried this branch with `--host_force_python=PY2` and looks like setting this to `PY3` gives us a slightly better error message:

> ERROR: /Users/user/rules_apple/test/starlark_tests/targets_under_test/ios/BUILD:290:16 Bundling app_with_alternate_icons failed: //tools/alticonstool:alticonstool: This target is being built for Python 2 but (transitively) includes Python 3-only sources. You can get diagnostic information about which dependencies introduce this version requirement by running the `find_requirements` aspect. If this is used in a genrule, you may need to migrate from tools to exec_tools. For more info see the documentation for the `srcs_version` attribute: https://docs.bazel.build/versions/master/be/python.html#py_binary.srcs_version caused by //tools/alticonstool:alticonstool

With `PY2AND3`, it would fail at `xctoolrunner`:

> ERROR: /Users/user/rules_apple/test/starlark_tests/targets_under_test/ios/BUILD:290:16: AssetCatalogCompile test/starlark_tests/targets_under_test/ios/app_with_alternate_icons-intermediates/xcassets failed: (Exit 1): xctoolrunner failed: error executing command bazel-out/host/bin/tools/xctoolrunner/xctoolrunner actool --compile ... (remaining 20 argument(s) skipped)
Traceback (most recent call last):
  File "/private/var/tmp/_bazel_user/c481fd6bdd7c10e4da1c8594f7656dc0/execroot/build_bazel_rules_apple/bazel-out/host/bin/tools/xctoolrunner/xctoolrunner.runfiles/build_bazel_rules_apple/tools/xctoolrunner/xctoolrunner.py", line 44, in <module>
    from build_bazel_rules_apple.tools.wrapper_common import execute
ImportError: No module named build_bazel_rules_apple.tools.wrapper_common
Note: The failure of target //tools/xctoolrunner:xctoolrunner (with exit code 1) may have been caused by the fact that it is a Python 3 program that was built in the host configuration, which uses Python 2. You can change the host configuration (for the entire build) to instead use Python 3 by setting --host_force_python=PY3.

> If this error started occurring in Bazel 0.27 and later, it may be because the Python toolchain now enforces that targets analyzed as PY2 and PY3 run under a Python 2 and Python 3 interpreter, respectively. See https://github.com/bazelbuild/bazel/issues/7899 for more information.
Target //test/starlark_tests/targets_under_test/ios:app_with_alternate_icons failed to build

Reply:
I guess the issue isn't in alticontool but rather in xctoolrunner. It's may be fine to make it `PY3` anyway, after all most tools are already.

Reply:
Ah well
```
py_binary(
    name = "xctoolrunner",
    srcs = ["xctoolrunner.py"],
    python_version = "PY3",
    visibility = [
        "//apple/internal:__pkg__",
    ],
    deps = [":xctoolrunner_lib"],
)
```